linera-views/src/views/map_view.rs

```diff
@@ -604,13 +640,13 @@ where
     /// assert_eq!(part_keys.len(), 2);
     /// # })
     /// ```
-    pub async fn for_each_key_value_while<F>(
-        &self,
+    pub async fn for_each_key_value_or_bytes_while<'a, F>(
+        &'a self,
         mut f: F,
         prefix: Vec<u8>,
     ) -> Result<(), ViewError>
     where
-        F: FnMut(&[u8], &[u8]) -> Result<bool, ViewError> + Send,
+        F: FnMut(&[u8], ValueOrBytes<'a, V>) -> Result<bool, ViewError> + Send,
```

Comment:
Why would a user want to see the bytes?

Reply:
Yes, wrong move. I wanted the test functions to be running, but that was a wrong idea.